apps/console/src/app/components/api-keys/PezzoApiKeyListItem.tsx

```diff
@@ -15,6 +16,11 @@ interface Props {
 export const PezzoApiKeyListItem = ({ value }: Props) => {
   const [copied, copy] = useCopyToClipboard();
 
+  const onCopy = () => {
+    copy(value);
+    trackEvent("organization_api_key_copy");
```

Comment:
```suggestion
    trackEvent("organization_api_key_copied");
```
To be consistent with other events, let's use past tense